deltaspike/modules/jsf/impl/src/test/java/org/apache/deltaspike/test/jsf/impl/injection/uc004/MyBeanValidator.java

```diff
@@ -26,15 +26,17 @@
 import jakarta.faces.validator.Validator;
 import jakarta.faces.validator.ValidatorException;
 import jakarta.inject.Inject;
+import jakarta.inject.Named;
 
-@FacesValidator("myBeanValidator")
+@Named
+@FacesValidator(value = "myBeanValidator", managed = true)
 public class MyBeanValidator implements Validator, StateHolder
 {
 
     @Inject
     private MyBean myBean;
 
-    private String validValue = "Apache";
+    private String validValue = "DeltaSpike";
```

Comment:
Can you please explain?

Reply:
All tests sent "DeltaSpike" as OK value:
```
src/test/java/org/apache/deltaspike/test/jsf/impl/injection/uc001/InjectionDroneTest.java:        convertedValue.sendKeys("DeltaSpike");
src/test/java/org/apache/deltaspike/test/jsf/impl/injection/uc002/InjectionDroneTest.java:        convertedValue.sendKeys("DeltaSpike");
src/test/java/org/apache/deltaspike/test/jsf/impl/injection/uc003/InjectionDroneTest.java:        convertedValue.sendKeys("DeltaSpike");
src/test/java/org/apache/deltaspike/test/jsf/impl/injection/uc004/InjectionDroneTest.java:        convertedValue.sendKeys("DeltaSpike");
```
Not one test sent "Apache".

Reply:
```
[INFO] ------< org.apache.deltaspike.modules:deltaspike-jsf-module-impl >------
[INFO] Building Apache DeltaSpike JSF-Module Impl 2.0.0-SNAPSHOT
[INFO]   from pom.xml
...
[INFO] Results:
[INFO] 
[INFO] Tests run: 115, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  59.339 s
[INFO] Finished at: 2024-02-25T23:24:46+01:00
[INFO] ------------------------------------------------------------------------
```

Reply:
Weird that it worked before